src/index.ts

```diff
@@ -90,15 +90,16 @@ internals.isEnabled = (namespace, level): boolean => {
 /**
  * @typedef {Function} createLogger
  * @param {String} [namespace]
+ * @param {boolean} canForceWrite
  * @return {Logger}
  */
-export const createLogger = (namespace?: string): Logger => {
+export const createLogger = (namespace?: string, canForceWrite?: boolean): Logger => {
```

Comment:
I don't think we need this parameter here, can I? The force logging maybe just affect only on `LogMethod`.

If it affect on the method `createLogger`, we have already the method `setLevel`

Reply:
We need this param for add a function in a LogMethod for all level